pyproject.toml

```diff
@@ -39,7 +39,7 @@ dependencies = [
     "toolz >= 0.10.0",
     "tornado >= 6.0.4",
     "urllib3 >= 1.24.3",
-    "zict >= 2.2.0",
+    "zict >= 3.0.0",
```

Comment:
lower zict versions raise for whatever reason if I pass a str key. I don't see a reason to not upgrade, though.

Maybe this issue is related to the perf regression I'm seeing but I'm not super motivated debugging spilling perf (it's just 10-20% not an order of magnitute)

Reply:
I'm very surprised about the regression in spilling. Particularly in the uncompressible test I would have expected disk I/O to be 2 to 3 orders of magnitude slower than the stringification.

Reply:
Shall we open an issue to investigate that and move forward with this PR?